app/services/evidence/rule_dsl.rb

```diff
@@ -47,15 +47,15 @@ def other(&block)
 
       def execute_client(crime_application)
         if @client_proc.respond_to?(:call)
-          @client_proc.call(crime_application, crime_application.applicant)
+          @client_proc.call(crime_application)
         else
```

Comment:
Using a submitted application means that rule logic can be simplified. As a result Applicant and Partner were rarely being used and have therefore been removed.